sklearn/preprocessing/data.py

```diff
@@ -719,6 +726,10 @@ class MaxAbsScaler(BaseEstimator, TransformerMixin):
     n_samples_seen_ : int
         The number of samples processed by the estimator. Will be reset on
         new calls to fit, but increments across ``partial_fit`` calls.
+
+    See also
+    --------
+    maxabs_scale: Equivalent function without the object oriented API
```

Comment:
Same as [this comment](https://github.com/scikit-learn/scikit-learn/pull/6748/files#r61788862)